credential-management/non-fully-active.https.html

```diff
@@ -42,28 +42,28 @@
     // Try to get credentials while not fully active...
     await promise_rejects_dom(
       t,
-      "NotAllowedError",
+      "InvalidStateError",
       DOMExceptionCtor,
       credentials.get({ signal }),
-      "Expected NotAllowedError for get() on non-fully-active document"
+      "Expected InvalidStateError for get() on non-fully-active document"
     );
 
     // Try to create credentials while not fully active...
     await promise_rejects_dom(
       t,
-      "NotAllowedError",
+      "InvalidStateError",
       DOMExceptionCtor,
       credentials.create({ signal }),
-      "Expected NotAllowedError for create() on non-fully-active document"
+      "Expected InvalidStateError for create() on non-fully-active document"
     );
 
     // Try to prevent silent access while not fully active...
     await promise_rejects_dom(
       t,
-      "NotAllowedError",
+      "InvalidStateError",
       DOMExceptionCtor,
       credentials.preventSilentAccess(),
-      "Expected NotAllowedError for preventSilentAccess() on non-fully-active document"
+      "Expected InvalidStateError for preventSilentAccess() on non-fully-active document"
```

Comment:
(should we do `store`? feels like the unloved child, since only chromium ever implemented something using it...)

Reply:
oh whoops, yes... let me send another PR for that.

Reply:
Ah, I forgot that I couldn't test that because we need to pass an instance of `BasicCredential` to store. 

In theory, we could test it with the Web Authn automation API. We currently don't implement that in WebKit, but probably should at some point 🤞.... let me file a bug and we figure out how to test it.

Reply:
Filed https://github.com/w3c/webappsec-credential-management/issues/251